server.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const http         = require('http');
+const finalhandler = require('finalhandler');
+const Router       = require('router');
+
+const router = new Router();
+
+router.get('/', (request, response) => {
+  // A good place to start!
+  response.end();
+});
+
+const server = http.createServer((request, response) => {
+  router(request, response, finalhandler(request, response));
+});
+
+exports.listen = function(port, callback) {
```

Comment:
They'll have covered the `exports` object in a previous unit, so maybe just reminding them that we're attaching `listen()` to that object?